unity/Assets/Scripts/PhysicsSceneManager.cs

```diff
@@ -567,6 +585,11 @@ List<SimObjType> excludedReceptacleTypes
                 foreach (SimObjPhysics receptacleSop in IterShuffleSimObjPhysicsDictList(objTypeToReceptacles, rng)) {
                     List<ReceptacleSpawnPoint> targetReceptacleSpawnPoints;
 
+                    if(receptacleSop.ContainedGameObjects().Count > 0) {
+                        //this object already has something in it, skip over it since we currently can't account for detecting bounds of a receptacle + any contained objects
+                        continue;
+                    }
```

Comment:
Does this mean we won't place more than 1 object on, e.g., a table?

Reply:
perhaps... I will double check this. The intent is to only filter out objects that are pickupable that also already contain some object. Those shouldn't be moved. So things like a countertop or a table should not be skipped over, but things like a plate or pot should.

Reply:
Once you've checked this, feel free to merge from my perspective.

Reply:
ok, i've updated this to only skip pickupable objects, and things are working as expected.